frontend/README.md

```diff
@@ -8,15 +8,15 @@ Create a .env file and populate it with this.
 
 ```conf
 NODE_ENV=development
-API_URL=http://backend:8080/
+API_URL=http://localhost:5000/
```

Comment:
not 100% sure if these configurations are used for non-development. If they are they can be reverted

Reply:
As said before - these are here since the initial dev setup that had the API running in docker containers (instead of locally in VS Code). Since then we have moved on from that. Unless @devinleighsmith has a different opinion, I'm okay with this change